app/related_things.rb

```diff
@@ -7,8 +7,8 @@ def initialize(manual, current_page)
   end
 
   def related_applications
```

Comment:
Do we want to rename this method to related_repos also? It can be used to refer to gems e.g
[govuk_publishing_components](https://github.com/alphagov/govuk-developer-docs/blob/d08e38bf969fba6875c95c9a0585e4469632b13f/source/manual/how-to-remove-jquery-from-javascript.html.md?plain=1#L8)

Reply:
Good spot 👍  I've pushed up a commit.